tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -652,6 +660,15 @@ Section Uninstall
     {{/each}}
   {{/each}}
 
+  ; Delete deep links
+  {{#each deep_link_protocol as |protocol| ~}}
+    ReadRegStr $DL0 SHCTX "Software\Classes\{{protocol}}\shell\open\command" ""
```

Comment:
What is `$DL0`? can't find any mention of it in NSIS docs

Reply:
afaik nsis doesn't care what it's called and i think it meant DeepLink0 for me (don't ask what the 0 is for). I can change it to $R7 if you prefer that. We use $R_ for other ReadRegStr calls and 0-6 is used already.

Reply:
> afaik nsis doesn't care what it's called

wild, didn't know that



> i think it meant DeepLink0 for me (don't ask what the 0 is for). I can change it to $R7 if you prefer that. We use $R_ for other ReadRegStr calls and 0-6 is used already.

$R7 is fine, also we can use any other variable, they are not used anymore after this call iirc.